.gitignore

```diff
@@ -55,7 +55,6 @@ depcomp
 /containers/*/rpms
 /cockpit.service
 /cockpit-wrapper
-/node_modules
```

Comment:
Does it hurt/conflict with a submodule to leave that in? I wonder what happens if you don't pull the submodule, and just run `npm install` directly manually. You may not be able to push that anywhere, but a local build still ought to work?

Reply:
I'm pretty sure this would conflict with the "always make sure `node_modules` matches the gitlink" logic that gets run every time we type `make`.  That particular piece of logic is *very* aggressive: it would delete your `node_modules` and replace it with the downloaded one.